File: application/src/main/java/application/ElevatorApplication.java
package application;

import application.model.Elevator;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ElevatorApplication {

    public static void main(String[] args) {
        SpringApplication.run(ElevatorApplication.class, args);
    }

}

File: application/src/main/kotlin/application/ElevatorApplication.kt
package application

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

//import org.springframework.boot.runApplication

@SpringBootApplication
open class ElevatorApplication

//    @JvmStatic
    fun main(args: Array<String>) {
        runApplication<ElevatorApplication>(*args)
    }
